photofiltre.txt

```diff
@@ -4,7 +4,7 @@ Version = 7.2.1
 License = Freeware
 Description = A complete image retouching program. This version comes with layers but is more limited than the Studio X version.
 Category = 3
-URLSite = http://www.photofiltre-studio.com/pf7-en.htm
+URLSite = http://202.129.240.234/Software/PhotoFiltre-7.2.1/pf7-setup-en-7.2.1.exe
```

Comment:
This replaces a valid page URL with a download URL, while the actual download URL is not working.